docs/user-guide/api-mediation/wto-message-on-startup.md

```diff
@@ -0,0 +1,103 @@
+# Configuring initial API Mediation Layer startup message for Syslog
+
+:::info Role: system programmer
+:::
+
+:::note Functionality of this feature is available from Zowe 2.18. 
+:::
+
+Startup of the API Mediation Layer can be configured to present a message in the Syslog that the API Mediation Layer is started and ready. This setup is typically used in combination with Workload Automation tools to manage the lifecycle of Zowe. 
```

Comment:
It's good to have ZWEAM001I, unless you have fixed something recently, ZWEAM001I does not mean APIML is ready, but does indicate it's started and maybe has no errors?

By the time you see ZWEAM001I message, the Desktop is usually already running and if you try login, which involves APIML, APIML can respond with an HTTP 405. If you keep retrying, eventually APIML is fully ready and succeeds. It's been a longstanding concern that is handled in zlux code https://github.com/zowe/zlux-server-framework/blob/v2.x/staging/plugins/sso-auth/lib/apimlHandler.js#L324-L326

I'm not sure whether this is due to some z/OSMF JWT stuff, or what, but I use ZWES1601I to indicate full APIML readiness because ZSS retries APIML until successful.